robot_skills/test/motion/arm-performance

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env python
```

Comment:
This script is unrelated to the grippersensor, right? maybe put in different branch.